sklearn/linear_model/ridge.py

```diff
@@ -847,8 +847,10 @@ def fit(self, X, y, sample_weight=None):
         -------
         self : Returns self.
         """
+        alphas = np.asarray(self.alphas)
```

Comment:
check https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/linear_model/ridge.py line 832.  The default argument for `init` was a numpy array.